.github/workflows/tests-integration.yml

```
@@ -0,0 +1,68 @@
+name: Tests (Integration)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

[Question] Do you expect to execute the CI jobs in the forked repositories?
If not we can stop it as follows:
```suggestion
    # This action cannot be executed in the forked repository.
    if: github.repository == 'optuna/optuna'
    steps:
    - name: Checkout
      uses: actions/checkout@v2
```

---

Thanks, fixed in https://github.com/optuna/optuna/pull/1352/commits/d18aa2a3d1a03c30d2f3a700dab084bb5f14ea8d.